google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionRunnerImpl.java

```diff
@@ -765,7 +762,7 @@ public void setSpan(Span span) {
         new Callable<T>() {
           @Override
           public T call() {
-            boolean useInlinedBegin = inlineBegin;
+            boolean useInlinedBegin = true;
             if (attempt.get() > 0) {
               if (useInlinedBegin) {
```

Comment:
Could we remove the branching here, since `useInlinedBegin` is always true?

Reply:
Yep, that's quite useless. Thanks for noticing.